manifest_base.yml

```diff
@@ -18,8 +18,6 @@ applications:
   health-check-type: process
   command: celery beat --app webservices.tasks
 - name: celery-worker
-  instances: 1
-  memory: 512M
```

Comment:
Note that this was accounted for in the subsequent commits - thanks @LindsayYoung! :-)